lib/src/accordion/accordion.component.ts

```diff
@@ -0,0 +1,149 @@
+import {
+    AfterContentInit,
+    Component,
+    EventEmitter,
+    HostBinding,
+    Input,
+    Output,
+    ViewEncapsulation
+} from '@angular/core';
+import { animate, AnimationEvent, state, style, transition, trigger } from '@angular/animations';
+import { anyToBoolean } from '../util';
+
+@Component({
+    selector: 'hc-accordion',
+    templateUrl: './accordion.component.html',
+    styleUrls: ['./accordion.component.scss'],
+    exportAs: 'hcAccordion',
+    animations: [
+        trigger('openState', [
+            state('open, open-instant', style({
+                height: '*'
+            })),
+            state('void', style({
+                height: '0px',
+                visibility: 'hidden'
+            })),
+            transition('void => open-instant', animate('0ms')),
+            transition('void <=> open', animate('400ms ease'))
+        ])
+    ],
+    encapsulation: ViewEncapsulation.None
+})
+export class AccordionComponent implements AfterContentInit {
+
+    private animationDisabled = false;
+    private currentlyAnimating = false;
+    private _hideToolbar = false;
+    private _isOpen = false;
+
+    @Input() triggerAlign: 'left' | 'right' = 'left';
+
+    @Input()
+    get hideToolbar(): boolean {
+        return this._hideToolbar;
+    }
+
+    set hideToolbar(hide) {
+        this._hideToolbar = anyToBoolean(hide);
+    }
+
+    @Input()
+    get open(): boolean {
+        return this.isOpen;
+    }
+
+    set open(opened: boolean) {
+        this.toggle(anyToBoolean(opened));
+    }
+
+    @Output() opened = new EventEmitter();
+
+    @Output() openStart = new EventEmitter();
+
+    @Output() closed = new EventEmitter();
+
+    @Output() closeStart = new EventEmitter();
+
+    get isOpen(): boolean {
+        return this._isOpen;
+    }
+
+    @HostBinding('class.hc-accordion') true;
+
```

Comment:
I'm getting a ts error on this line 
`[ts] Member 'true' implicitly has an 'any' type.`